server/socket.js

```diff
@@ -50,6 +51,7 @@ export default function startSocketServer(http) {
 
 
     socket.on(CS.JOIN_TO_CHANNEL, channelId => {
+      if (!validate(channelId).isString().inRange(20, 30).end()) return;
       joinToChannel(socket.sessionId, channelId, (userId) => {
         socket.join(channelId);
```

Comment:
What happens if a user tries to join some channel with non-existing id? How bad things would be?